qt/scientific_interfaces/ISISReflectometry/Reduction/ValidateLookupRow.h

```diff
@@ -23,30 +24,27 @@ namespace ISISReflectometry {
     the lookup table on the ExperimentSettings tab are valid.
  */
 
-template <typename T> using ValidatorT = boost::optional<T>;
```

Comment:
There are still some instances where we use a `std::optional` to indicate validity. I wonder if it would be good to retain `ValidatorT` (under a better name?) so we are clear in these cases.

Reply:
I'm not really fussed. I don't like aliases at all to be honest!

In this case I think there could be a benefit to differentiate between the different use of optional's. The variable name's should suffice though.